### ISSUE TITLE
small file using Xlink by internal ID - errors

### ISSUE THREAD

**Issue by porosnie** [eurocontrol-swim/aixm-graph#12]
I have upload a file that is using only internal links (by gml:Id). The file does not seem to be processed correctly. The internal direct links (for example, from AirportHeliport to OrganisationAuthority) do not seem to be recognised. Also, I get an error message when trying to download the skeleton file. The errors also occur with the small extract attached.
[EAD-LEMD.xml.zip](https://github.com/eurocontrol-swim/aixm-graph/files/4672079/EAD-LEMD.xml.zip)

**Comment by antavelos** [comment]
The non recognition of the internal link is related with the fact that elements with xlink:href vary in the prefix of the href value. The prefixes I have identified so far are:
```
urn:uuid:
urn:uuid.
uuid.
#
```

This file however uses a another prefix: `#uuid.`. I'll include this one as well, but is there somewhere a list with all the possible prefixes or is this arbitrary?

**Comment by porosnie** [comment]
There are only two prefixes: "#" which indicates an internal link and "urn:uuid:" which indicates an abstract linl. 

When "#" is used, all the rest is part of the ID value and should match identical. For example, when the following is encountered:

(1) xlink:href="#uuid.09424934-7e4b-0ae9-e053-0b751dc2b779", the link is to the element that has gml:id strictly equal to "uuid.09424934-7e4b-0ae9-e053-0b751dc2b779"

(2) xlink:href="#ID_1234", the link is to the element that has gml:id strictly equal to "ID_1234"

When "urn:uuid:" is used, the rest shall matcj the gml:identifier of the target feature, not to its gml:id attribute. For example:

(3) xlink:href="urn:uuid:09424934-7e4b-0ae9-e053-0b751dc2b779", the link is to the element that has gml:identifer with codeSpace="urn:uuid:" and its value equal to "09424934-7e4b-0ae9-e053-0b751dc2b779"

I have attached a file with both cases.

[EAD-LEMD-LEMM.zip](https://github.com/eurocontrol-swim/aixm-graph/files/4677708/EAD-LEMD-LEMM.zip)

**Comment by antavelos** [comment]
So, cases like the below ones, found in DONLON, are supposed to be a typo?

`xlink:href="uuid.0942496b-7ce7-0b22-e053-0b751dc2499f"`
`xlink:href="urn:uuid.3618ff4a-13c0-451b-b510-27ea9e716c0f"`

**Comment by porosnie** [comment]
Yes, these are typing errors. I will add an issue to the Donlon maintenance site in gitHub: https://github.com/aixm/donlon
